### PR TITLE
V3.19.0

### PR DESCRIPTION
**Pull request by SC426**
[Upstream](https://github.com/marshmallow-code/marshmallow)
[Changelog](https://github.com/marshmallow-code/marshmallow/blob/dev/CHANGELOG.rst)

### Actions
- Updated version number and sha256
- Updated build section
- Updated dependencies and tests
- Linter fixes

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,49 +1,54 @@
...
+    - python
+    - packaging >=17.0
 
 test:
   requires:
+    - pip
     - pytest
     - simplejson
     - pytz
     - python-dateutil
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]: Doing a quick grep in GH, it doesn't look like this is required anymore...

Comment (SC426) [reply recipe/meta.yaml]: I think we always want to run pip check in tests for our purposes, so pip is needed for that reason.

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml]: I'm pointing at `python-dateutil`.

Comment (SC426) [reply recipe/meta.yaml]: Ok I see that here now https://github.com/marshmallow-code/marshmallow/blob/5a10e83c557d2ee97799c2b85bec49fc90381656/setup.py#L5

It's been removed. Thanks for catching that.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**